process/smartContract/process.go

```diff
@@ -787,6 +787,25 @@ func (sc *scProcessor) computeBuiltInFuncGasUsed(
 func (sc *scProcessor) ExecuteBuiltInFunction(
 	tx data.TransactionHandler,
 	acntSnd, acntDst state.UserAccountHandler,
+) (vmcommon.ReturnCode, error) {
+	sw := core.NewStopWatch()
+	sw.Start("executeBuiltIn")
+	returnCode, err := sc.doExecuteBuiltInFunction(tx, acntSnd, acntDst)
+	sw.Stop("executeBuiltIn")
+	duration := sw.GetMeasurement("executeBuiltIn")
+
+	if duration > executeDurationAlarmThreshold {
+		log.Debug(fmt.Sprintf("scProcessor.ExecuteBuiltInFunction(): execution took > %s", duration), "sc", tx.GetRcvAddr(), "duration", duration, "returnCode", returnCode, "err", err, "data", string(tx.GetData()))
```

Comment:
executeDurationAlarmThreshold instead duration in %s, as the duration is displayed in "duration"